### DISCUSSION TITLE
Leave the state field blank

### DISCUSSION THREAD

**Discussion by aunco** [woocommerce/woocommerce#52164] [Checkout Flow]
Currently the state field gets the value from the shop address when default customer location is set to shop country/region. The problem is most customers are not in the same state with the shop. Please leave the state field blank by default and let the customer enter the value.

**Comment by aunco** [comment]
If the above cannot be done, please remove the state from the "country / state" field on store address setting page.
![Screenshot 2024-10-19 132623](https://github.com/user-attachments/assets/fc08c607-74d6-4ee5-9818-138ee1cf1fb0)

**Comment by alexflorisca** [comment]
Hi @aunco thank you for pointing that out. Some stores may not have the majority of customers in the same state as the store. However the customer would have to change the state anyway, even if its not populated. There are stores where this feature is beneficial. For example ones with a physical presence that also operate online. Could the `geolocate` option better serve you here?
